### PR TITLE
Adds haml as template engine

### PR DESCRIPTION
**Pull request by GeorgeJLewis**
### What we have learned

- Refreshed memory on adding haml as template engine.  

### What we did
- Adds haml-rails gem to gem file
- Converts application layout file to haml

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'turbolinks', '~> 5'
 gem 'bootsnap', '>= 1.1.0', require: false
+gem "haml"
```
Comment (tochman) [inline Gemfile]: There’s a Rails version of this gem. https://github.com/indirect/haml-rails Perhaps it’s better suited for you?

**Review by tochman — COMMENTED** [review]
See my comment

File: Gemfile (hunk 1 above)
Comment (GeorgeJLewis) [reply Gemfile]: I saw this but was not sure about it i will take a look now.

**Review by oliverochman — COMMENTED** [review]
You guys need to convert the layout.html.erb file to a haml file

**Review by oliverochman — APPROVED** [review]
LGTM!

**Pull request merged**